Makefile

```diff
@@ -27,8 +27,12 @@ all: bootstrap
 config:
 ifeq (riscv,$(ARCH))
 	@echo $(RISCV_EXEC) > $(OUT)/target
+	@echo \#define TARGET_RISCV 1 > $(SRCDIR)/def.h
```

Comment:
`def.h` is quite confusing since we already have file `defs.h`.

Reply:
We can generate the file `config.h` in the top-level directory rather than `$(SRCDIR)`.

Reply:
It would be similar to the way what autotools do.

Reply:
how about `target.h`?

Reply:
`target.h` is confusing because it usually refers to target-specific configurations and entries. See LLVM for example.

Reply:
OK, let's use `config.h`